### PR TITLE
Privacy policy testing

### PR DESCRIPTION
**Pull request by alexevic**
Test Automation for Privacy Policy page. Page content is tested from .txt file in data folder.

### CODE REVIEW

**Review by KietasRiestutelis — APPROVED** [review]

File: src/page-objects/vision.page.ts [hunk 1]
```diff
@@ -1,7 +1,6 @@
-import { VisionData } from '../data/vision.data';
 import urls from '../data/urls';
-import { clickElement, getElement, getElementText } from '../helpers/elements';
```
Comment (Deimokas) [inline src/page-objects/vision.page.ts:3]: are these changes related to privacy policy page?

**Review by Deimokas — APPROVED** [review]
